Formula/c/crunchy-cli.rb

```diff
@@ -0,0 +1,32 @@
+class CrunchyCli < Formula
+  desc "Command-line downloader for Crunchyroll"
+  homepage "https://github.com/crunchy-labs/crunchy-cli"
+  url "https://github.com/crunchy-labs/crunchy-cli/archive/refs/tags/v3.0.3.tar.gz"
+  sha256 "9ee633b0de18b3cc2b6246c596b6b752a888e56203b8bc59db10dd7a8e4fa70a"
+  license "MIT"
+
+  head "https://github.com/crunchy-labs/crunchy-cli.git", branch: "master"
+
+  depends_on "rust" => :build
+  depends_on "ffmpeg"
+  depends_on "openssl@3"
+
+  on_linux do
+    depends_on "pkg-config" => :build
+  end
+
+  def install
+    system "cargo", "install", "--no-default-features", "--features", "openssl-tls", *std_cargo_args
+    man1.install Dir["target/release/manpages/*"]
+    bash_completion.install "target/release/completions/crunchy-cli.bash"
+    fish_completion.install "target/release/completions/crunchy-cli.fish"
+    zsh_completion.install "target/release/completions/_crunchy-cli"
```

Comment:
Does the executable have a function to generate these completions?

Reply:
I do not think so. I believe it is generated within the build process.

Reply:
I will mention this upstream.